docs/source/models/concept_models/vivarium_mncnh_portfolio/mncnh_interventions.rst

```diff
@@ -0,0 +1,130 @@
+
+Vivarium Modeling Strategy
+--------------------------
+
+We will use the enhanced decision tree below to find the probability of RDS mortality with and without access to CPAP that are logically consistent with the baseline delivery facility rates and baseline CPAP coverage (and population level RDS mortality probability). Since the neonatal model include timesteps for both the early and late neonatal periods, this  alculation must be calculated for both time periods.
+
+We will then add an attribute to each simulant indicating whether the birth occurs at a facility with access to CPAP (which will be dependent on the facility choice, i.e. no access for home births and lower access for BEmONC than CEmONC facilities).
+
+We will then use the conditional probabilities for simulants with and without access to determine which simulants die from RDS.
+
+.. todo::
+
+  include the relative risk of RDS mortality with CPAP here and in the concept model document, without duplicating it (see recent PR to add quickstart docs in the readme for an example of how)
+
+
+Calibration Strategy
+--------------------
+
+The following decision tree shows all of the paths from delivery facility choice to CPAP availability.  Each path corresponds to a disjoint event that we can sum over to find the population probability of RDS mortality.
+
+.. graphviz::
+
+    digraph CPAP {
+        rankdir = LR;
+        start [color="none"]
+        facility [label="1"]
+        home [label="p_RDS_without_CPAP"]
+        BEmONC [label="7"]
+        CEmONC [label="7"]
+        BEmONC_wo [label="p_RDS_without_CPAP"] 
+        BEmONC_w [label="p_RDS_with_CPAP"]
+        CEmONC_wo [label="p_RDS_without_CPAP"] 
+        CEmONC_w [label="p_RDS_with_CPAP"]
+
+        start -> facility  [label = "delivery facility type"]
+        facility -> home  [label = "home birth"]
+        facility -> BEmONC  [label = "BEmONC"]
+        facility -> CEmONC  [label = "CEmONC"]
+
+        BEmONC -> BEmONC_w  [label = "CPAP available"]
+        BEmONC -> BEmONC_wo  [label = "CPAP unavailable"]
+
+        CEmONC -> CEmONC_w  [label = "CPAP available"]
+        CEmONC -> CEmONC_wo  [label = "CPAP unavailable"]
+    }
+
+.. math::
+    \begin{align*}
+        p_\text{RDS} 
+        &= \sum_{\text{paths without CPAP}} p(\text{path})\cdot p_\text{RDS_wo}\\
+        &+ \sum_{\text{paths with CPAP}} p(\text{path})\cdot p_\text{RDS_w}\\[.1in]
+        p_\text{RDS_w} &= RR_\text{CPAP} \cdot p_\text{RDS_wo}
+    \end{align*}
+
+
+
+When we fill in the location-specific values for delivery facility rates, CPAP coverage, relative risk of mortality with CPAP access, and mortality probability (which is also age-specific), this becomes a system of two linear equations have two unknown (p_RDS_w and p_RDS_wo), which we can solve analytically.
+
+Scenarios
```

Comment:
I'm not sure I understand the purpose of this section

Reply:
> I'm not sure I understand the purpose of this section

Is this section where we would describe how to model the CPAP intervention in different scenarios, or would that go in the concept model?

Reply:
I was thinking that this would include the general approach to scenarios, e.g. increase coverage to different levels in different types of health facilities, and then the specific values would go in the concept model.